Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
+    path_expander (1.1.1)
+    pg (1.5.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+      method_source (~> 0.9.0)
     pry-byebug (3.3.0)
       byebug (~> 8.0)
       pry (~> 0.10)
-    pry-rails (0.3.5)
-      pry (>= 0.9.10)
-    pry-rescue (1.4.5)
+    pry-rails (0.3.9)
```

Comment:
### [pry-rails](https://github.com/rweng/pry-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.3.5 → 0.3.9

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/pry-rails/0.3.5/0.3.9)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/pry/pry-rails/compare/v0.3.5...v0.3.9).

These are the first 10 commits:
- (a8693e7) [Bump minimum required pry version to 0.10.4](https://github.com/pry/pry-rails/commit/a8693e7ee7e06ca264f6f11b482f034ce24a6a35)
- (59f8d86) [Removed wrong `/` in pry-everywhere link.](https://github.com/pry/pry-rails/commit/59f8d8686467b5269baec37723aeeea29527c65d)
- (f964db1) [Merge pull request #91 from mobilutz/patch-1](https://github.com/pry/pry-rails/commit/f964db1535e5fba00d15020aa30ee9714ea2a106)
- (2d632cb) [Change show-routes command to allow multiple grep options](https://github.com/pry/pry-rails/commit/2d632cb7499792806e2549518f6dc688f52e658f)
- (bac26a5) [Merge pull request #92 from ar-shestopal/Alex-Shestopal-Show…](https://github.com/pry/pry-rails/commit/bac26a5257ba3ffe6dadd6ebafca9b326e84ccb1)
- (c3d0fa9) [Merge pull request #90 from cjlarose/bump-required-pry-versi…](https://github.com/pry/pry-rails/commit/c3d0fa9e3cc11fddc016e7cdc6a56d383a53c1f0)
- (098f2f1) [Fix show-models spec for Rails 5.0](https://github.com/pry/pry-rails/commit/098f2f15415226622b2e4272ddfd62155255fb4a)
- (dd3e429) [Fix show-routes issues](https://github.com/pry/pry-rails/commit/dd3e4298243ee1e6233c78e498e7e9151c7e5f93)
- (2533b68) [Bump version to 0.3.6](https://github.com/pry/pry-rails/commit/2533b68c677ef13900c5d78db68b1271b74fcd55)
- (be97a21) [Fix compatibility of specs with Mongoid and Rails 5.1](https://github.com/pry/pry-rails/commit/be97a215a4143f8a50c91a9ab481ee43cc9406cd)


</details>